gbpservice/nfp/orchestrator/modules/device_orchestrator.py

```diff
@@ -83,12 +84,16 @@ def __init__(self, conf, controller):
                                   }
 
     def _log_event_created(self, event_id, event_data):
-        LOG.info(_LI("Device Orchestrator, RPC Handler, Created event "
-                     "%s(event_name)s with event data: %(event_data)s"),
-                 {'event_name': event_id, 'event_data': event_data})
+        log_meta_data = (event_data.get('log_meta_data')
+                         if 'log_meta_data' in event_data else '')
```

Comment:
As we are doing get on event_data dict, no need to check for key, we can also do
event_data.get('log_meta_data', '')